code/game/objects/items/devices/radio/radio.dm

```diff
@@ -309,7 +312,11 @@
 			if (idx && (idx % 2) == (message_mods[RADIO_EXTENSION] == MODE_L_HAND))
 				return
 
+<<<<<<< HEAD
 	talk_into(speaker, raw_message, , spans, language=message_language, direct=FALSE) //SKYRAT EDIT CHANGE - GUNPOINT
```

Comment:
```suggestion
	talk_into(speaker, raw_message, , spans, language=message_language, message_mods=filtered_mods, direct=FALSE) //SKYRAT EDIT CHANGE - GUNPOINT
```
and the uh merge conflict will still be there
just helping out because I want this merrrrrged